### ISSUE TITLE
Cannot run qsiprep. Missing dependencies: fugue, prelude

### ISSUE THREAD

**Issue by suzhc** [PennLINC/qsiprep#407]
Hi @mattcieslak ,
When I am using qsiprep:0.15.3, I got an error like this:
```
Cannot run qsiprep. Missing dependencies:
        fugue (Interface: FUGUE)
        fugue (Interface: FUGUE)
        prelude (Interface: PRELUDE)
```
0.15.2 also had this problem to me. But 0.14.3 is fine.
This is my command:
```
docker run -ti --rm \
-v working-dir/qsiprep-proj/:/data:ro \
-v working-dir/qsiprep-proj/derivatives/:/out \
-v working-dir/qsiprep-proj/working/:/working \
-v working-dir/qsiprep-proj/code/license.txt:/opt/freesurfer/license.txt \
pennbbl/qsiprep:0.15.3 \
/data /out participant \
--participant_label 00000192 \
--unringing-method mrdegibbs \
--fs-license-file /opt/freesurfer/license.txt \
--nthreads 64 \
--omp-nthreads 64 \
--output_resolution 1.2 \
-w /working \
--stop-on-first-crash 
```
And There is no log in my log dir, this is my log dir:
```
logs/
├── CITATION.html
├── CITATION.md
└── CITATION.tex
```

**Comment by mattcieslak** [comment]
I suggest running the docker or singularity version of qsiprep. All the dependencies come installed in the image, with versions that are known to work

**Comment by estephanmoana** [comment]
Hello, I am having this same issue with qsiprep version 0.16.0RC3 as a singularity container. Any suggestions on how to address this? Thank you.

**Comment by mattcieslak** [comment]
This is solved in 0.16.1 and later, sorry for not updating the issue
